draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -199,52 +201,32 @@ The following RateLimit response header fields are defined.
 
 ## RateLimit {#ratelimit-field}
 
-A server uses the "RateLimit" response header field to communicate its quota policies.
-
-The field is a Dictionary. The allowed keys are defined in the "Hypertext Transfer Protocol (HTTP) RateLimit Keywords and Parameters Registry", as described in {{iana-ratelimit-parameters}}.
+A server uses the "RateLimit" response header field to communicate the client quota consumption state.
 
-The following Keys are defined in this specification:
+The field is expressed as a Structured Fields {{STRUCTURED-FIELDS}} sf-Item that identifies the policy and its associated consumption state as parameters.
 
-  limit:
-  :  The REQUIRED "limit" key value conveys
-     the expiring limit ({{ratelimit-limit-keyword}}).
-  remaining:
-  :  The OPTIONAL "remaining" key value conveys
-     the remaining quota units ({{ratelimit-remaining-keyword}}).
-  reset:
-  : The REQUIRED "reset" key value conveys
-     the time window reset time ({{ratelimit-reset-keyword}}).
+An origin server MUST not return a response that contains multiple rate limit fields with the same policy value.
 
-This specification does not define Parameters for this field.
-If they appear, they MUST be ignored.
-
-This field cannot appear in a trailer section.
+The field is a Dictionary. The allowed keys are defined in the "Hypertext Transfer Protocol (HTTP) RateLimit Keywords and Parameters Registry", as described in {{iana-ratelimit-parameters}}.
 
-## Limit Keyword {#ratelimit-limit-keyword}
+The following parameters are defined in this specification:
 
-The "limit" keyword indicates the [service limit](#service-limit) associated with the client in the current [time window](#time-window). If the client exceeds that limit, it MAY not be served.
+  r:
+  :  This parameter value conveys the remaining quota units for the identified policy ({{ratelimit-remaining-parameter}}).
 
-It is an Item and its value is a non-negative Integer referred to as the "expiring limit".
-This specification does not define Parameters for it.
-If they appear, they MUST be ignored.
+  t:
+  : This OPTIONAL parameter value conveys the time window reset time for the identified policy ({{ratelimit-reset-parameter}}).
```

Comment:
In some places the document says "t" is a parameter (e.g. here it says "optional parameter") and in other places it says it's a keyword (e.g. below in the name of the "Reset Keyword" section). Since "r" is called a parameter, I think it makes sense to call "t" a parameter consistently as well :v:

Reply:
Yes. As we switched from sf-dictionary to sf-item with parameters we needed to move from the term "keyword" to "parameter". I will check to make sure we have removed all uses of keyword.